test/feature-benchmark/mzcompose.py

```diff
@@ -134,20 +135,21 @@ def start_services(
         print(f"The version of the '{instance.upper()}' Mz instance is:")
         c.run("materialized", "--version")
 
-        c.start_and_wait_for_tcp(services=["materialized"])
-        c.wait_for_materialized()
+        c.up("materialized")
+        c.wait_for_materialized(port=6875)
```

Comment:
What's up with the `port=6875` here? Is there a bug somewhere?

Reply:
The materialized service exposes two ports by default, and wait_for_materialized waits for both. However, in the single-binary Mz , if one is used for benchmarking, the second port is not ever open so the wait_for_materialized fails.